metasyn/config.py

```diff
@@ -97,6 +97,15 @@ def from_toml(cls, config_fp: Union[str, Path]) -> MetaConfig:
         meta_config:
             A fully initialized MetaConfig instance.
         """
+        if Path(config_fp).suffix == '':
+            raise FileNotFoundError(f"It appears '{config_fp}' is not a valid filepath."
+                                    f" Please provide a path to a .toml file to load a MetaConfig"
+                                    f" from.")
```

Comment:
I think we should do it the other way around: try to load the file and if it doesn't load, show the errors. In the try/catch you should catch the error and reraise it. So if you get a FileNotFoundError, reraise from there. Otherwise you probably get some kind of Parse error. In that case you can check for the suffix and give a hint depending on that.

Reply:
I've changed it - is this what you had in mind?

Reply:
Yes, thanks! The only thing I would split is that there can be parse errors because the .toml file is broken or a non-toml file is supplied. Perhaps we can split the error message between the two with the suffix test that you made before?

Reply:
Done! Feel free to suggest a change to the phrasing.